### ISSUE TITLE
Unable to view flutter module in Android

### ISSUE THREAD

**Issue by athompson887** [flutter/put-flutter-to-work#193]
I have built the aar.
I have built the Android project and it runs and shows the list of items.
When the list is scrolled it seems to try to load something but nothing actually renders.
Investigation showed that it was compaining about 👎 
Cannot call this method in a scroll callback. Scroll callbacks mightbe run during a measure & layout pass where you cannot change theRecyclerView data. Any method call that might change the structureof the RecyclerView or the adapter contents should be postponed tothe next frame.
I tried to mitigate this by starting the module after a delay after onResume - same sitation with error
Rejecting re-init on previously-failed class java.lang.Class<io.flutter.embedding.android.FlutterActivity$1>

Am I on my own here.  I cannot get a flutter module to actually load in an android project.

**Comment by domesticmouse** [comment]
Can I get a hand here @gaaclarke and @xster?

**Comment by athompson887** [comment]
Hi Brett,

Thanks for responding to me so quickly.  I am an app developer and I have asked by my work to investigate adding flutter modules to native android and the medium post looks fantastic and pretty straight forward but as I have said it doesn’t seem to work properly.  I have not changed the grade version or anything like that - the code is pretty much as it has been written.  I am using Electric Eel but have tried it on older IDE’s.

Also the instructions for adding a flutter module are no longer available in more recent versions of the Android Studio IDE and have not been available for some time.

I am using a non intel Mac but I don’t think that is the cause because I know I have tried iton a PC.  

I really appreciate any help you can give.

Kind regards

Andrew

**Comment by domesticmouse** [comment]
While I wait on support from the engineers, can I ask you to do two things? 

  - Can you raise a bug on the [`flutter/flutter` issue tracker](https://github.com/flutter/flutter/issues) regarding the missing instructions?
  - Ask a question along these lines on the [Flutter Stack Overflow](https://stackoverflow.com/questions/tagged/flutter)? 

I will continue to chase up this information, but having a bug tracking the specific missing documentation helps, and you'll get great support from the Flutter extended community on Stack Overflow.

**Comment by elainedb** [comment]
Hi Andrew, Brett,

I took a look at the project and I was able to fix it for Android :) I opened a PR: #195

Andrew, you can check out the code and check that it's working for you.

Once it's merged, it would be nice to update the documentation with the latest assets (the ones provided on the Readme have expired)

**Comment by athompson887** [comment]
Brilliant thank you Sent from my iPhoneOn 28 Feb 2023, at 15:11, elainedb ***@***.***> wrote:﻿
Hi Andrew, Brett,
I took a look at the project and I was able to fix it for Android :) I opened a PR: #195
Andrew, you can check out the code and check that it's working for you.
Once it's merged, it would be nice to update the documentation with the latest assets (the ones provided on the Readme have expired)

**Comment by athompson887** [comment]
Hi,It works!  I am so pleased.  It is a credit to you that you responded so quickly.  Thank you so much.Kind regards Andrew Sent from my iPhoneOn 28 Feb 2023, at 15:28, Andrew Thompson ***@***.***> wrote:﻿Brilliant thank you Sent from my iPhoneOn 28 Feb 2023, at 15:11, elainedb ***@***.***> wrote:﻿
Hi Andrew, Brett,
I took a look at the project and I was able to fix it for Android :) I opened a PR: #195
Andrew, you can check out the code and check that it's working for you.
Once it's merged, it would be nice to update the documentation with the latest assets (the ones provided on the Readme have expired)
